pkg/v7/configmap/desired_test.go

```diff
@@ -147,470 +145,41 @@ func Test_ConfigMap_GetDesiredState(t *testing.T) {
-							label.Organization: "giantswarm",
-							label.ServiceType:  "managed",
-						},
-					},
-					Data: map[string]string{
-						"values.json": differentSettingsJSON,
-					},
-				},
-				&corev1.ConfigMap{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "kube-state-metrics-values",
-						Namespace: metav1.NamespaceSystem,
-						Labels: map[string]string{
-							label.App:          "kube-state-metrics",
-							label.Cluster:      "5xchu",
-							label.ManagedBy:    "cluster-operator",
-							label.Organization: "giantswarm",
-							label.ServiceType:  "managed",
-						},
-					},
-					Data: map[string]string{
-						"values.json": "{\"image\":{\"registry\":\"quay.io\"}}",
-					},
-				},
-				&corev1.ConfigMap{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "net-exporter-values",
-						Namespace: metav1.NamespaceSystem,
-						Labels: map[string]string{
-							label.App:          "net-exporter",
-							label.Cluster:      "5xchu",
-							label.ManagedBy:    "cluster-operator",
-							label.Organization: "giantswarm",
-							label.ServiceType:  "managed",
-						},
-					},
-					Data: map[string]string{
-						"values.json": "{\"namespace\":\"kube-system\"}",
-					},
-				},
-				&corev1.ConfigMap{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "node-exporter-values",
-						Namespace: metav1.NamespaceSystem,
-						Labels: map[string]string{
-							label.App:          "node-exporter",
-							label.Cluster:      "5xchu",
-							label.ManagedBy:    "cluster-operator",
-							label.Organization: "giantswarm",
-							label.ServiceType:  "managed",
-						},
-					},
-					Data: map[string]string{
-						"values.json": "{\"image\":{\"registry\":\"quay.io\"}}",
-					},
-				},
-			},
-		},
-		{
-			name: "case 3: ingress controller already migrated",
-			configMapConfig: ClusterConfig{
-				APIDomain:  "5xchu.aws.giantswarm.io",
-				ClusterID:  "5xchu",
-				Namespaces: []string{},
-			},
-			configMapValues: ConfigMapValues{
-				ClusterID:                         "5xchu",
-				IngressControllerMigrationEnabled: true,
-				Organization:                      "giantswarm",
-				WorkerCount:                       3,
-			},
-			ingressControllerReleasePresent: true,
-			expectedConfigMaps: []*corev1.ConfigMap{
-				&corev1.ConfigMap{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "cert-exporter-values",
-						Namespace: metav1.NamespaceSystem,
-						Labels: map[string]string{
-							label.App:          "cert-exporter",
-							label.Cluster:      "5xchu",
-							label.ManagedBy:    "cluster-operator",
-							label.Organization: "giantswarm",
-							label.ServiceType:  "managed",
-						},
-					},
-					Data: map[string]string{
-						"values.json": "{\"namespace\":\"kube-system\"}",
-					},
+			providerChartSpecs: []key.ChartSpec{},
+			expectedConfigMapSpecs: []ConfigMapSpec{
+				{
+					Name:      "cert-exporter-values",
+					Namespace: metav1.NamespaceSystem,
```

Comment:
Now we just test that the configmap names  and namespaces are correct.

EDIT. This comment was in the wrong place.